hack/actions/check-uncommitted-codegen.sh

```diff
@@ -24,5 +24,6 @@ if git status -s ${TARGETS[@]} 2>&1 | grep -E -q '^\s+[MADRCU]'
 then
 	echo Uncommitted changes in generated sources:
 	git status -s ${TARGETS[@]}
+	git diff
```

Comment:
```suggestion
	git --no-pager diff
```

just to make sure the full output is printed and you don't get stuck in a pager

Reply:
besides, LGTM

Reply:
> just to make sure the full output is printed and you don't get stuck in a pager

Besides, LGTM